src/block/block_ext.c

```diff
@@ -1082,7 +1107,7 @@ __block_merge(
         }
         WT_RET(__block_off_remove(session, block, el, before->off, &ext));
 
-        __wt_verbose_debug2(session, WT_VERB_BLOCK,
+        __wt_verbose(session, WT_VERB_BLOCK,
           "%s: range grows from %" PRIdMAX "-%" PRIdMAX ", to %" PRIdMAX "-%" PRIdMAX, el->name,
```

Comment:
This feels like a level above default to me. For example if we discard a block we'll already get the level 1 debug message:
`[WT_VERB_BLOCK][DEBUG_1]: block free 0: 11096064/28672`
Then this log should print:
`[WT_VERB_BLOCK][DEBUG_2]: live.discard: range grows from 10665984-11096064, to 10665984-11124736`

This second message is extra information that should be included in the next level of verbosity.

Reply:
The same goes for the other log messages.

Reply:
Although I'm wondering if the other messages that you pointed out that were not on the same level should be changed to level 2.

Reply:
@Sean04  
I have adjust the log verbose level, please review again, thanks.
:)